edgelet/iotedge/src/config/apply.rs

```diff
@@ -638,6 +641,10 @@ mod tests {
 
             let test_name = case_directory.file_name().unwrap().to_str().unwrap();
 
+            if test_name.eq("manifest-trust-bundle") {
```

Comment:
Delete the `manifest-trust-bundle` in test-files/ rather than doing this.

Reply:
done